content/2016.12.how-to-develop-a-webapp-with-Kinto.md

```diff
@@ -0,0 +1,247 @@
+Title: How to develop a WebApp with Kinto
+Date: 2016-12-01 10:20
+Slug: develop-a-chat-webapp-with-kinto
+Authors: Enguerran Colson, Frank Taillandier, Kinto Team
+Summary: An example of how Kinto handles a webapp's data storage.
+
+_This post was [originally published in french on Enguerran Colson's blog](https://blog.ticabri.com/2016/11/29/developper-une-webapp-avec-kinto/)._
+
+## Our goal
+
+Hi folks, today, we are gonna create a simple WebApp that allows visitors to chat in a room accessible from a public URL.
+
+![KintoChat, our instant chat WebApp](images/kintochat-fullscreen.png)
+{.image}
+
+* A really simple chat webapp with minimal UI.
+* We will need a bit of JavaScript to enhance and animate interactions within the application and we will also need a way to store and share the app's data. Ideally, we would like our application to consume a REST API exposing endpoints like `https://api.discute.re/v1/rooms/12345/messages`, where `discute.re` is the domain of our application (*discutere* means to chat in Latin).
+
+Our API endpoints would be:
+
+* `/rooms` to return all chat rooms,
+* `/rooms/:id/` to return a particular chat room identified by an ID,
+* `/rooms/:id/messages` to return all messages from a particular room.
```

Comment:
This is the original CSS for blog posts, I may suggest the same and it will be good to change the font family as I find the one used very unreadable.

We may change style after the publication of some articles. We could get back some content from servicedenuage.fr